ebi_eva_common_pyutils/assembly_utils.py

```diff
@@ -27,6 +28,18 @@
 logger = log_cfg.get_logger(__name__)
 
 
+def is_patch_assembly(assembly_accession: str) -> bool:
+    """
+    Check if a given assembly is a patch assembly
+    Please see: https://www.ncbi.nlm.nih.gov/grc/help/patches/
+    """
+    xml_root = download_xml_from_ena(f'https://www.ebi.ac.uk/ena/browser/api/xml/{assembly_accession}')
+    xml_assembly = xml_root.xpath("//ASSEMBLY_ATTRIBUTE[TAG='count-patches']/VALUE")
+    if len(xml_assembly) == 0:
+        return False
+    return int(xml_assembly[0].text) > 0
```

Comment:
Solution looks good. My only objection (not major at all) is that we have an indiscriminate mixture of reliance on ENA and NCBI which will make it hard to say that we get our assembly information from NCBI or ENA.
I know they both provide the same information but I would feel more comfortable using a single source.